mmdet/models/detectors/maskformer.py

```diff
@@ -111,13 +116,12 @@ def simple_test(self, imgs, img_metas, **kwargs):
             img_metas (list[dict]): List of image information.
 
         Returns:
-            list[dict[str, np.array | tuple]]: Semantic segmentation \
-                results and panoptic segmentation results for each \
-                image.
-
+            list[dict[str, np.array | tuple[list]] | tuple[list]]:
+                Semantic segmentation results and panoptic segmentation \
+                results of each image for panoptic segmentation, or formatted \
+                bbox and mask results of each image for instance segmentation.
             .. code-block:: none
-
-                [
+                [# panoptic segmentation
```

Comment:
There are problems on rendering docs of this part, and need to fix it. Please change to 

```text
        Returns:
            list[dict[str, np.array | tuple[list]] | tuple[list]]:
                Semantic segmentation results and panoptic segmentation \
                results of each image for panoptic segmentation, or formatted \
                bbox and mask results of each image for instance segmentation.
            
            .. code-block:: none
                
                [
                    # panoptic segmentation
                    {
                        'pan_results': np.array, # shape = [h, w]
                        'ins_results': tuple[list],
                        # semantic segmentation results are not supported yet
                        'sem_results': np.array
                    },
                    ...
                ]

            or

            .. code-block:: none
            
                [
                    # instance segmentation
                    (
                        bboxes, # list[np.array]
                        masks # list[list[np.array]]
                    ),
                    ...
                ]

```

Reply:
Adjusted.